serialization/json_ld/examples/package.jsonld

```diff
@@ -26,7 +26,7 @@
           "algorithm": "sha1",
           "hashValue": "71c4025dd9897b364f3ebbb42c484ff43d00791c"
         },
-                {
```

Comment:
I think this change should not be mixed with other changes, as it is very prone to cause merge conflicts